src/rng.js

```diff
@@ -0,0 +1,5 @@
+import crypto from 'crypto';
+
+export default function nodeRNG() {
```

Comment:
```suggestion
export default function() {
```

Look for `Math.getRandomValues()` here, as above?

Reply:
Let's wait with functional changes for a follow-up PR.